PowerShellForGitHub.psd1

```diff
@@ -1,229 +1,234 @@
-        'Set-GitHubMilestone',
-        'Set-GitHubProfile',
-        'Set-GitHubProject',
-        'Set-GitHubProjectCard',
-        'Set-GitHubProjectColumn',
-        'Set-GitHubRepository'
-        'Set-GitHubRepositoryTopic',
-        'Split-GitHubUri',
-        'Test-GitHubAssignee',
-        'Test-GitHubAuthenticationConfigured',
-        'Test-GitHubOrganizationMember',
-        'Test-GitHubRepositoryVulnerabilityAlert',
-        'Unlock-GitHubIssue'
-    )
-
-    AliasesToExport = @(
-        'Delete-GitHubComment',
-        'Delete-GitHubIssueComment',
-        'Delete-GitHubLabel',
-        'Delete-GitHubMilestone',
-        'Delete-GitHubProject',
-        'Delete-GitHubProjectCard',
-        'Delete-GitHubProjectColumn'
-        'Delete-GitHubRepository',
-        'Get-GitHubBranch',
-        'Get-GitHubComment',
-        'New-GitHubAssignee',
-        'New-GitHubComment',
-        'Remove-GitHubComment',
-        'Set-GitHubComment',
-        'Transfer-GitHubRepositoryOwnership'
-        'Update-GitHubIssue',
-        'Update-GitHubLabel',
-        'Update-GitHubCurrentUser',
-        'Update-GitHubRepository'
-    )
-
-    # Cmdlets to export from this module
-    # CmdletsToExport = '*'
-
-    # Variables to export from this module
-    # VariablesToExport = '*'
-
-    # Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
-    PrivateData = @{
-
-        PSData = @{
-
-            # Tags applied to this module. These help with module discovery in online galleries.
-            Tags = @('GitHub', 'API', 'PowerShell')
-
-            # A URL to the license for this module.
-            LicenseUri = 'https://aka.ms/PowerShellForGitHub_License'
-
-            # A URL to the main website for this project.
-            ProjectUri = 'https://aka.ms/PowerShellForGitHub'
-
-            # A URL to an icon representing this module.
-            # IconUri = ''
-
-            # ReleaseNotes of this module
-            ReleaseNotes = 'https://github.com/microsoft/PowerShellForGitHub/blob/master/CHANGELOG.md'
-        }
-    }
-
-    # Default prefix for commands exported from this module. Override the default prefix using Import-Module -Prefix.
-    # DefaultCommandPrefix = 'GH'
-
-    # Modules that must be imported into the global environment prior to importing this module
-    # RequiredModules = @()
-
-    # Assemblies that must be loaded prior to importing this module
-    # RequiredAssemblies = @()
-
-    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
-    # ScriptsToProcess = @()
-
-    # List of all modules packaged with this module
-    # ModuleList = @()
-
-    # List of all files packaged with this module
-    # FileList = @()
-
-    # HelpInfo URI of this module
-    # HelpInfoURI = ''
-}
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@TylerLeonhardt - Can you take a quick look at this file?  It looks like the whole file changed.  Likely an encoding issue.  Would be great if you could get it updated in the PR such that it reflects just the changed lines.  Might require you to revert the file and then re-apply the changes.  I've noticed that using the GitHub web resolver does this to the file.

Reply:
fixed